app/knowledge_panels.py

```diff
@@ -65,13 +65,13 @@ async def hunger_game_kp(self):
             url, des = val
             html.append(
                 {
-                    "id": id,
+                    "id": int(id),
                     "element_type": "text",
-                    "text_element": {"html": f"<p><a href='{url}'>{t_description} {des}</a></p>"},
+                    "text_element": f"<p><a href='{url}'>{t_description} {des}</a></p>",
                 },
```

Comment:
@alexgarel I change the response to make it more unified!

Reply:
:+1: